charts/airbyte/values.yaml

```diff
@@ -1,8 +1,10 @@
 ## @section Global Parameters
 
+## @param global.imageRegistry Global Docker image registry
 ## @param global.storageClass Global StorageClass for Persistent Volume(s)
 ##
 global:
+  imageRegistry: ""
```

Comment:
curious: what does this default to?

Reply:
Having `global.imageRegistry: ""` will use dockerhub.

Example of the image generated using the default:

```
helm template airbyte . -s templates/scheduler/deployment.yaml | grep "image:" 
        image: airbyte/scheduler:0.29.21-alpha
```

Example of overriding this to use a private registry:

```
helm template airbyte . --set global.imageRegistry="my.docker.registry" -s templates/scheduler/deployment.yaml | grep "image:"
        image: my.docker.registry/airbyte/scheduler:0.29.21-alpha
```